Index: application/Controllers/admin.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Controller for the Admin side of things.\nfrom flask import render_template\n\nfrom application import app\n\n\n@app.route(\"/admin\")\n@app.route(\"/admin/home\")\ndef admin_home():\n    return render_template(\"admin/home.html\")\n\n\n@app.route(\"/admin/login\")\ndef admin_login():\n    return render_template(\"admin/login.html\")\n\n\n@app.route(\"/admin/register\")\ndef admin_register():\n    return render_template(\"admin/register.html\")\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/application/Controllers/admin.py b/application/Controllers/admin.py
--- a/application/Controllers/admin.py	(revision 36cccd94bf9b4c4b690ba8591de601334ed2ccf0)
+++ b/application/Controllers/admin.py	(date 1640085456278)
@@ -1,11 +1,17 @@
 # Controller for the Admin side of things.
 from flask import render_template
 
+from flask import Flask, render_template, request, redirect, url_for
+from application.Forms import CreateUserForm, CreateCustomerForm
+import shelve
+from application.Models.User import User
+from application.Models.Customer import Customer
+
 from application import app
 
 
 @app.route("/admin")
-@app.route("/admin/home")
+# @app.route("/admin/home")
 def admin_home():
     return render_template("admin/home.html")
 
Index: application/Models/FPUser.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/application/Models/FPUser.py b/application/Models/FPUser.py
new file mode 100644
--- /dev/null	(date 1640085456330)
+++ b/application/Models/FPUser.py	(date 1640085456330)
@@ -0,0 +1,7 @@
+# New user class for FoodyPulse specifically.
+
+class FPUser:
+    count_id = 0
+
+    def __init__(self):
+        pass
\ No newline at end of file
